app/src/main/java/kth/jjve/xfran/HomeScreenActivity.java

```diff
@@ -20,6 +32,27 @@ protected void onCreate(Bundle savedInstanceState) {
         FrameLayout contentFrameLayout = findViewById(R.id.content_frame);
         getLayoutInflater().inflate(R.layout.act_home, contentFrameLayout);
 
+        /*----- HOOKS -----*/
+        WODTitle = findViewById(R.id.home_WODTitle);
+        WODDescription = findViewById(R.id.home_WODDescription);
+        WODExercises = findViewById(R.id.home_WODExercises);
+        Username = findViewById(R.id.home_username);
+        PlannedWorkouts = findViewById(R.id.home_rv);
+
+        /*-----  VM  -----*/
+        homeVM = ViewModelProviders.of(this).get(HomeVM.class);
+        homeVM.init(getApplicationContext());
+        homeVM.getUserName().observe(this, s -> Username.setText(s));
+        homeVM.getWODTitle().observe(this, s -> WODTitle.setText(s));
+        homeVM.getWODDescription().observe(this, s -> WODDescription.setText(s));
+        homeVM.getWODExercises().observe(this, ls -> {
+            StringBuilder exercises = new StringBuilder();
+            for (String s : ls){
+                exercises.append(s).append("\n");
+            }
+            WODExercises.setText(exercises);
+        });
```

Comment:
we use this quite a number of times so in my pull request I made it a method, we can change it here after we merge those :)